superset/config.py

```diff
@@ -1582,7 +1582,7 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
 CONTENT_SECURITY_POLICY_WARNING = True
 
 # Do you want Talisman enabled?
-TALISMAN_ENABLED = utils.cast_to_boolean(os.environ.get("TALISMAN_ENABLED", True))
+TALISMAN_ENABLED = utils.cast_to_boolean(os.environ.get("TALISMAN_ENABLED", False))
```

Comment:
@Henry-Do-Su @SachinJoshiTest @NareshReddy553 Nakul mentioned he want to discuss on this with you. Could you please suggest if this is okay by disabling the CSP? 

``` TALISMAN_ENABLED defaults to True; set this to False in order to disable CSP```